apps/token-manager/app/src/screens/Details.js

```diff
@@ -0,0 +1,193 @@
+import React, { useCallback } from 'react'
+import {
+  Accordion,
+  Bar,
+  BackButton,
+  Box,
+  Button,
+  GU,
+  Split,
+  formatTokenAmount,
+  textStyle,
+  useLayout,
+  useTheme,
+} from '@aragon/ui'
+import SummaryBar from '../components/SummaryBar'
+import SummaryRows from '../components/SummaryRows'
+import VestingInfoBoxes from '../components/VestingInfoBoxes'
+import { useAppLogic, toISODate, useVestedTokensInfo } from '../app-logic'
+import { format, formatDistanceStrict, parseISO } from 'date-fns'
+
+const formatDate = date => `${format(date, 'do MMM yyyy, HH:mm O')}`
+
+function Details({ tokenSymbol, selectHolder, vestings }) {
+  const theme = useTheme()
+  const { selectedHolder } = useAppLogic()
+  const handleBack = useCallback(() => selectHolder(-1), [selectHolder])
```

Comment:
Note that here we're doing an implicit cast from `-1` to `'-1'`, and we may want to use a different mechanism for this.

Regardless of whether we change the unselected state to be `null` or a falsey value, we may want to expose a `unselectHolder` function from `useAppLogic()`.